bigbluebutton-html5/private/config/settings.yml

```diff
@@ -33,6 +33,7 @@ public:
     copyright: '©2023 BigBlueButton Inc.'
     html5ClientBuild: HTML5_CLIENT_VERSION
     helpLink: https://bigbluebutton.org/html5/
+    delayForUnmountOfSharedNote: 120000
```

Comment:
Could you please add an inline comment stating what this is (and what it means to reduce/increase the value; also -- milliseconds?)